scripts/check_3pp_licenses.js

```diff
@@ -1,280 +0,0 @@
-// *****************************************************************************
-// Copyright (C) 2021 Ericsson and others
-//
-// This program and the accompanying materials are made available under the
-// terms of the Eclipse Public License v. 2.0 which is available at
-// http://www.eclipse.org/legal/epl-2.0.
-//
-// This Source Code may also be made available under the following Secondary
-// Licenses when the conditions for such availability set forth in the Eclipse
-// Public License v. 2.0 are satisfied: GNU General Public License, version 2
-// with the GNU Classpath Exception which is available at
-// https://www.gnu.org/software/classpath/license.html.
-//
-// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
-// *****************************************************************************
-// @ts-check
-
-const cp = require('child_process');
-const fs = require('fs');
-const path = require('path');
-const readline = require('readline');
-const kSECRET = Symbol('secret');
-
-// Submit any suspicious dependencies for review by the Eclipse Foundation, using dash-license "review" mode?
-var autoReviewMode = process.argv.includes('--review');
-const project = process.argv.find(arg => /--project=(\S+)/.exec(arg)?.[1]) ?? 'ecd.theia';
-
-const NO_COLOR = Boolean(process.env['NO_COLOR']);
-const dashLicensesJar = path.resolve(__dirname, 'download/dash-licenses.jar');
-const dashLicensesSummary = path.resolve(__dirname, '../dependency-check-summary.txt');
-const dashLicensesBaseline = path.resolve(__dirname, '../dependency-check-baseline.json');
```

Comment:
This file is checked in, can we remove it now? I think it’s not referenced in the new config and it’s empty.